browser/resources/settings/email_aliases_page/content/email_aliases_modal.tsx

```diff
@@ -143,9 +143,15 @@ export const DeleteAliasModal = ({
   alias: Alias,
   emailAliasesService: EmailAliasesServiceInterface
 }) => {
-  const onDeleteAlias = () => {
-    emailAliasesService.deleteAlias(alias.email)
-    onReturnToMain()
+  const [deleteErrorMessage, setDeleteErrorMessage] =
+    React.useState<string | null>(null)
+  const onDeleteAlias = async () => {
+    const { errorMessage } = await emailAliasesService.deleteAlias(alias.email)
```

Comment:
should we be disabling the buttons while we're deleting? Otherwise we could queue a bunch of delete requests at once

also, should we clear the error message before sensing the next delete command?


```suggestion
    setDeleteErrorMessage(null)
    const { errorMessage } = await emailAliasesService.deleteAlias(alias.email)
```

Reply:
good idea; done